tasks/stages/update-demo

```diff
@@ -1,5 +1,8 @@
 #! /bin/bash
 
+set -euo pipefail
+shopt -s inherit_errexit nullglob
```

Comment:
What's this for? 🤡

Reply:
Did some research and decided to settle down on that configuration [based on this article](https://dev.to/banks/stop-ignoring-errors-in-bash-3co5).

Reply:
My goal is to catch as many errors as possible, hence that additional line (described in the article above, too).

Reply:
Thanks for sharing knowledge.